src/edfi-performance-test/edfi_performance_test/performance_tester.py

```diff
@@ -24,51 +24,55 @@ def prepare_for_output(args: MainArguments) -> str:
     output_base_path = f"{args.output}/pipeclean"
     return output_base_path
 
+def spawn_pref_tests(args: MainArguments) -> None:
+    # setup Environment and Runner
+    env = Environment(user_classes=[DummyUser], host=args.baseUrl)
+    env.create_local_runner()
 
-async def run(args: MainArguments) -> None:
-
-    try:
-        logger.info("Starting performance test...")
-        start = time.time()
+    # start a WebUI instance
+    env.create_web_ui("127.0.0.1", 8089)
```

Comment:
Can we also get a switch to turn this off? That was the other half of this ticket. Or just remove the web_ui instead of having a switch.

Reply:
I will just remove. I never saw it opening web ui and it is not useful unless tests are being run from there.